integrade/tests/api/v1/test_cloud_accounts.py

```diff
@@ -0,0 +1,279 @@
+"""Tests for cloud accounts.
+
+:caseautomation: automated
+:casecomponent: api
+:caseimportance: high
+:caselevel: integration
+:requirement: Cloud Meter
+:testtype: functional
+:upstream: yes
+"""
+from urllib.parse import urljoin
+
+import pytest
+
+from integrade import api
+from integrade import config
+from integrade.tests.api.v1 import urls
+from integrade.tests.api.v1.utils import (
+    create_user_account,
+    get_auth,
+)
+from integrade.utils import gen_password, uuid4
+from integrade.utils import needs_valid_cloud_accounts
+
+
+@pytest.mark.serial_only
+@needs_valid_cloud_accounts
+def test_create_cloud_account(drop_account_data):
+    """Ensure cloud accounts can be registered to a user.
+
+    :id: d0174576-9b7c-48f7-8556-b560badf062d
+    :description: Ensure an user can register a cloud account by specifying
+        the role ARN.
+    :steps: 1) Create a user and authenticate with their password
+        2) Send a POST with the cloud account information to 'api/v1/account/'
+        3) Send a GET to 'api/v1/account/' to get a list of the cloud accounts
+        4) Attempt to create a duplicate and expect it to be rejected
+        5) Attempt to delete the account and expect to be rejected
+    :expectedresults: The server returns a 201 response with the information of
```

Comment:
If you want to be more explicit you can add a list to the expectedresults that each expected results item maps to items on the steps.

For example https://github.com/cloudigrade/integrade/blob/master/integrade/tests/api/v1/test_user_accounts.py#L88-L96